config/example.toml

```diff
@@ -11,14 +11,18 @@ db_url = "mysql://root:dev_web3_proxy@127.0.0.1:13306/dev_web3_proxy"
 # read-only replica useful when running the proxy in multiple regions
 db_replica_url = "mysql://root:dev_web3_proxy@127.0.0.1:13306/dev_web3_proxy"
 
+# This is the sweeper contract to deposit
```

Comment:
wait. we shouldn't be hard coding a sweeper contract here. we should be putting the factory in the config and then we can get the user's owned copy of the sweeper contract from there

Reply:
ah yes this is outdated, i removed it now